src/test/java/org/graylog/integrations/notifications/types/SlackEventNotificationTest.java

```diff
@@ -0,0 +1,167 @@
+package org.graylog.integrations.notifications.types;
+
+import com.floreysoft.jmte.Engine;
+import com.google.common.collect.ImmutableList;
+import org.graylog.events.notifications.*;
+import org.graylog.events.notifications.types.HTTPEventNotificationConfig;
+import org.graylog2.notifications.Notification;
+import org.graylog2.notifications.NotificationImpl;
+import org.graylog2.notifications.NotificationService;
+import org.graylog2.plugin.Message;
+import org.graylog2.plugin.MessageSummary;
+import org.graylog2.plugin.system.NodeId;
+import org.graylog2.shared.bindings.providers.ObjectMapperProvider;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class SlackEventNotificationTest extends SlackPluginTestFixture {
+
+    @Mock
+    NodeId mockNodeId;
+    private SlackEventNotification slackEventNotification;
```

Comment:
Use `@InjectMocks` so you don't need to manually build the test object in a `@Before` method

```suggestion
    @InjectMocks
    private SlackEventNotification slackEventNotification;
```